elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -40,15 +39,14 @@ import {
 	P2PResponsePacket,
 } from './p2p_types';
 import {
-	connectAndFetchPeerInfo,
 	constructPeerIdFromPeerInfo,
 	EVENT_BAN_PEER,
 	EVENT_CLOSE_INBOUND,
 	EVENT_CLOSE_OUTBOUND,
 	EVENT_CONNECT_ABORT_OUTBOUND,
 	EVENT_CONNECT_OUTBOUND,
 	EVENT_DISCOVERED_PEER,
-	EVENT_FAILED_PEER_INFO_UPDATE,
+	EVENT_FAILED_TO_COLLECT_PEER_DETAILS_ON_CONNECT,
```

Comment:
Like mentioned in the above comment, it will look better if we can break it down in two events `EVENT_FAILED_TO_FETCH_STATUS` and `EVENT_FAILED_TO_FETCH_PEERS`, maybe we can use `EVENT_FAILED_TO_FETCH_PEER_INFO ` from below for `EVENT_FAILED_TO_FETCH_STATUS`